app/models/exercise.rb

```diff
@@ -89,6 +90,9 @@ class Exercise < ActiveRecord::Base
 
   belongs_to :vocab_term, touch: true
 
+  scope :can_release_to_a15k, -> { where(release_to_a15k: true) }
+  scope :not_released_to_a15k, -> { where(a15k_identifier: nil) }
+
```

Comment:
it looks like these are never used independently, and are chained together in the export.  If that's the case why not make a:
```ruby
scope :needs_released_to_a15k, -> { where(release_to_a15k: true, a15k_identifier:nil ) }
```

Reply:
That is a good point, and I went to make the change.... but the one thing I like about it is for readability from the export code.  That code we will share with other members, and I like pointing out that we explicitly mark what we want to share and track if it has been shared yet.  So I may leave it.